portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -1,20 +1,95 @@
 ---
-page_title: Accounts | Portafly
-page_title_description: Page title of the Accounts page
-body_title: Accounts
-body_title_description": Body title of the Accounts page
-accounts_table:
-  aria_label: Accounts list
-  aria_label_description: Aria label of the Accounts table component
-  empty_state: There are no accounts
-  empty_state_description: Message of the Accounts table empty state
-  col_group: Group/org.
-  col_group_description: Title of the Group/org. column
-  col_admin: Admin
-  col_admin_description: Title of the Admin column
-  col_signup: Signup date
-  col_signup_description: Title of the Signup date column
-  col_apps: Apps
-  col_apps_description: Title of the Apps column
-  col_state: State
-  col_state_description: Title of the State column
+# Account settings > Users
+
+title_page: Accounts | Team members
+title_page_description: Title for the page to manage team members
+
+# Top area
+
+body_title: Team members
+body_title_description: Title for the main area to manage team members
+invite_new_member_button: Invite a new team member
+invite_new_member_button_description: Button link to send invitations to new team members
+
+# Main area
+
+## Tabs
+listing_tab: Listing
+listing_tab_description: Label for the tab that contains the list of all team members, selection criteria and actions that can be applied on them.
+invitations_tab: Invitations
+invitations_tab_description: Label for the tab that contains the list of all invitations, selection criteria and actions that can be applied on them.
+
+## Tab: Listing
+## Toolbar
+bulk_selector: {{number}} selected
+bulk_selector_description: Label for the bulk selector to indicate the number of chosen team members. This label is visible only when team members were selected.
+# Delete button: available in shared.yml
+team_members_filter: [Name]
+team_members_filter_description: List containing the criteria to filter team members
+[PROPOSED] team_members_typeahead: Filter by name
+team_members_typeahead_description: Filter by typing ahead the text
+
+team_members_table:
+  aria_label: Team members table
+  aria_label_description: Aria label of the table of team members
+  [PROPOSED] empty_state: Currently, there are no team members. Add team members.
```

Comment:
Thanks for the suggestion @damianpm . We'll discuss this option in our next standup.

Reply:
@porueesq I would take this as an option, since brackets can't be used this way in YAML (they are reserved for arrays). It has to me moved in a comment o written w/o them like this:
```suggestion
  PROPOSED empty_state: Currently, there are no team members. Add team members.
```

Reply:
@josemigallas We'll be using `#---PROPOSED` at the end of the line. Ideally, you should receive files with all the strings approved, so this temporary tag will be removed by when you start coding.